packages/map/src/components/Legend/components/LegendGroup/Legend.Group.tsx

```diff
@@ -0,0 +1,130 @@
+import { useContext, useMemo } from 'react'
+import './Legend.Group.css'
+import LegendShape from '@cdc/core/components/LegendShape'
+import { toggleLegendActive } from '@cdc/map/src/helpers/toggleLegendActive'
+import ErrorBoundary from '@cdc/core/components/ErrorBoundary'
+import ConfigContext from '../../../../context'
+
+interface LegendItem {
+  color: string
+  label: string
+  disabled?: boolean
+  special: boolean
+}
+
+interface GroupedData {
+  [key: string]: LegendItem[]
+}
+
+const LegendGroup = ({ legendItems }) => {
+  const { runtimeLegend, setAccessibleStatus, setRuntimeLegend, state, currentViewport } = useContext(ConfigContext)
+
+  const getGridColClass = (viewport: string) => {
```

Comment:
Instead of using the viewport can you use bootstraps helpers?

col-number, col-md-number, col-lg-number?